pkg/remotewrite/config.go

```diff
@@ -21,8 +21,10 @@ const (
 	defaultMetricPrefix = "k6_"
 )
 
+//nolint:gochecknoglobals
```

Comment:
mmm...should we make it not global? @oleiade what is your opinion? do you have a suggestion? I would avoid splitting it from the consts. Should we create a `defaultConfig` function?

Reply:
Let's keep it as it is and I will try to fix in my next refactor

Reply:
I have unresolved this as a reminder.